### PR TITLE
Added Microsoft OpenJDK 11 Preview version 11.0.10.9.

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/10174)

### CODE REVIEW

File: manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Moniker: ms-openjdk-11
+Tags:
+- Java
+- OpenJDK
+- "11"
+- LTS
+PackageUrl: https://www.microsoft.com/openjdk
+License: GPL2 with Classpath Exception.
+LicenseUrl: https://openjdk.java.net/legal/gplv2+ce.html
+MinimumOSVersion: 10.0.0.0
```
Comment (brunoborges) [inline manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml:16]: Will `winget` work with other versions of Windows? Or only 10?

File: manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Publisher: Microsoft
+PackageName: Microsoft OpenJDK 11 Preview
+Author: Microsoft
+ShortDescription: The Microsoft Build of OpenJDK is a new no-cost long-term supported distribution and Microsoft"s new way to collaborate and contribute to the Java ecosystem.
+Moniker: ms-openjdk-11
+Tags:
+- Java
+- OpenJDK
+- "11"
+- LTS
```
Comment (brunoborges) [inline manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml:12]: This package is not LTS. This tag should be removed.

File: manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Microsoft.OpenJDK.11
+Publisher: Microsoft
+PackageName: Microsoft OpenJDK 11 Preview
```
Comment (brunoborges) [inline manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml:4]: The name must be `Microsoft Build of OpenJDK 11 (Preview)`

File: manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+MinimumOSVersion: 10.0.0.0
+PackageVersion: 11.0.10.9
+InstallerType: msi
+Installers:
+- Architecture: x64
+  InstallerUrl: https://aka.ms/download-jdk/microsoft-jdk-11.0.10.9-windows-x64.msi
+  InstallerSha256: 4F9F6C9CD1CC9F1D6D893E59DD74FFC7711892CADE528937CD3ACB12E2B2DC5B
+  Scope: machine
+  InstallerLocale: en-US
+  ProductCode: "{8650B43A-3575-46E9-852B-47A35C3CA85B}"
```
Comment (brunoborges) [inline manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml:25]: Where did this code come from?

**Review by brunoborges — CHANGES_REQUESTED** [review]
Provided comments in the PR.

File: manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml (hunk 1 above)
Comment (denelon) [reply manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml:16]: Currently, only Windows 10.

Comment (brunoborges) [reply manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml:16]: Ok got it.

**Comment by brunoborges** [comment]
On a more general comment, I believe all input data should be "quoted". Because... YAML...

File: manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml (hunk 2 above)
Comment (jedieaston) [reply manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml]: Fixed.

File: manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml (hunk 3 above)
Comment (jedieaston) [reply manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml]: Fixed.

**Comment by jedieaston** [comment]
The schema validator hasn't been requiring quotes, @denelon do you have an opinion? I can fix it real quick if I need to.

**Comment by brunoborges** [comment]
I mean... I'm fine with strings not being quoted. But then.. it's YAML! :D

LGTM!

**Comment by denelon** [comment]
If it ain't broke, don't fix it is my general opinion. I've gone back and forth between all strings quoted. If the client works with the manifest as is, I'm fine. We may change guidance in the future, but I don't want to block on them :)

**Comment by denelon** [comment]
@jedieaston FYI: @brunoborges is our resident MSFT Java expert :)

**Review by ghost — APPROVED** [review]
Validation has completed

**Comment by denelon** [comment]
@jedieaston

https://twitter.com/richturn_ms/status/1379557808136318978

**Pull request merged**